samples/boot/oauth2login/src/main/resources/application.yml

```diff
@@ -21,14 +21,15 @@ security:
       github:
         client-id: your-app-client-id
         client-secret: your-app-client-secret
-        user-info-name-attribute-key: "name"
+        user-name-attribute-name: "name"
```

Comment:
While `user-name-attribute-name` is an option, we shouldn't list it for each of these because they should be defaulted for us. We know a reasonable default value for github, facebook, etc.